src/lib/registration/ui/import_certificate_dialog.rb

```diff
@@ -89,7 +82,7 @@ def import_dialog_content
         hide_help = displayinfo["TextMode"] && displayinfo["Width"] < 105
 
         window_height = displayinfo["Height"]
-        window_height = 25 if window_height > 25
+        window_height = 26 if window_height > 26
```

Comment:
Why 26? Just curious.

Reply:
Because in the default Qt installation it was just not enough, you had to scroll a bit. :wink: